nvm.sh

```diff
@@ -1044,6 +1044,20 @@ nvm_install_node_source() {
     make='gmake'
     MAKE_CXX="CXX=c++"
   fi
+
+  if [ "_$NVM_OS" = "_linux" ]; then
```

Comment:
It occurs to me that rather than checking the `NVM_OS`, this might be more reliable as just `if nvm_has nproc` etc - ie, assume if it has the binary, that it will work?